src/CMakeLists.txt

```diff
@@ -143,6 +143,14 @@ if(APPLE AND (CMAKE_CXX_COMPILER_ID STREQUAL "AppleClang" OR CMAKE_CXX_COMPILER_
         list(REMOVE_DUPLICATES CMAKE_CXX_IMPLICIT_LINK_LIBRARIES)
 endif()
 
+# Check for base threading library
+set(CMAKE_THREAD_PREFER_PTHREAD TRUE)
+find_package(Threads REQUIRED)
+# Fail if returned library is not pthread
+if(NOT CMAKE_USE_PTHREADS_INIT)
+  message(FATAL_ERROR "POSIX threads: not found")
+endif()
+
```

Comment:
Hm, yes, but where does `libdarktable` recieve `CMAKE_THREAD_LIBS_INIT`?
I think you want to add here:
```
list(APPEND LIBS ${CMAKE_THREAD_LIBS_INIT})
```